include/af/backend.h

```diff
@@ -24,6 +24,40 @@ extern "C" {
 AFAPI af_err af_set_backend(const af_backend bknd);
 #endif
 
+#if AF_API_VERSION >= 37
+/**
+   \param[in] lib_path is the path of the custom arrayfire library to
```

Comment:
This needs a brief description at the top.

Reply:
Thanks, I forgot about that since the function declarations around it didn't have brief descriptions as well.